tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -4730,7 +4760,8 @@ uint32_t SPIRVEmitter::processIntrinsicMul(const CallExpr *callExpr) {
     }
   }
 
-  emitError("Unsupported arguments passed to mul() function.");
+  emitError("unsupported arguments passed to mul() function",
```

Comment:
invalid argument type passed to mul intrinsic function?

Reply:
done